### PR TITLE
chore(ui-react): lint test files

### PR DESCRIPTION
**Pull request by calebpollman**
#### Description of changes
More test file linting updates

**Before**
Screen Shot 2023-01-24 at 1 29 53 PM

**After**
Screen Shot 2023-01-24 at 1 27 59 PM

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react/src/hooks/__tests__/useDeprecationWarning.test.tsx [hunk 1]
```diff
@@ -1,52 +1,55 @@
 import { renderHook } from '@testing-library/react-hooks';
 import { useDeprecationWarning } from '../useDeprecationWarning';
 
-const originalWarn = console.warn;
+// add empty mockImplementation to prevent logging output to the console
+const consoleWarnSpy = jest.spyOn(console, 'warn').mockImplementation();
```
Comment (reesscot) [inline packages/react/src/hooks/__tests__/useDeprecationWarning.test.tsx:5]: thank you!

File: packages/react/src/primitives/utils/testUtils.ts [hunk 2]
```diff
@@ -21,3 +23,12 @@ ${message}
     throw new Error((error as Error).message);
   }
 };
+
+export const testFlexProps: FlexContainerStyleProps = {
```
Comment (reesscot) [inline packages/react/src/primitives/utils/testUtils.ts:27]: nice!

**Review by reesscot — COMMENTED** [review]
LGTM, thanks for cleaning these up!

**Review by reesscot — APPROVED** [review]

**Review by wlee221 — APPROVED** [review]

**Review by zchenwei — APPROVED** [review]
LGTM

**Pull request merged**